tests/test-hwinfo.c

```diff
@@ -270,5 +297,7 @@ main (int   argc,
 
   g_test_add_func ("/hwinfo/cpu/current", test_get_cpu_info_for_current_system);
```

Comment:
Isn't this test now redundant with `test_get_computer_hwinfo` which makes the same assertions?

Reply:
The test cases have different idea and target.
* `test_get_cpu_info_for_current_system` checks the returned CPU information.
* `test_get_computer_hwinfo` checks the RAM, disk and CPU combination and they should be in order.

Reply:
I suppose so, but the latter one does do all the checks that the former does (assert_cpu_info_for_current_system() + implicitly eins_hwinfo_get_cpu_info()) – there is no case where `test_get_cpu_info_for_current_system` would fail and `test_get_computer_hwinfo` would pass.

I'm OK with leaving both tests though. They do check slightly different things and are not redundant the other way around: `/hwinfo/cpu/current` could pass while `/hwinfo/computer/current` fails, which would help narrow down the failure.